### PR TITLE
💅🏽  461 [New UX] Trace View # 3 (Test Results)

### PR DESCRIPTION
**Pull request by xoscar**
### Note: this is the first pass for the test results drawer. We'll be waiting for the final designs to fully implement the changes.

This PR moves the test results from the span detail/tab section to the bottom drawer.

## Changes

- Connects the test results component with the resizable drawer.

## Fixes

- (Partially) https://github.com/kubeshop/tracetest/issues/461

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [x] added a test

https://www.loom.com/share/432e43f773d64935b8d7e67de18445fb

### CODE REVIEW

File: web/cypress/integration/Trace/ShowTrace.spec.ts [hunk 1]
```diff
@@ -21,7 +21,6 @@ describe('Show Trace', () => {
     cy.get('[data-cy=span-details-attributes]').should('be.visible');
     cy.get('[data-cy=empty-assertion-table]').should('be.visible');
 
-    cy.get('[id*=tab-test-results').click();
-    cy.get('[data-cy=test-results]').should('be.visible');
+    cy.get('[data-cy=test-results]').should('exist');
```
Comment (xoscar) [inline web/cypress/integration/Trace/ShowTrace.spec.ts:24]: The test results its only partially displayed when the trace view loads

File: web/src/App.less [hunk 2]
```diff
@@ -24,12 +24,6 @@ body {
   background: #fbfbff;
 }
 
-.ant-typography {
```
Comment (xoscar) [inline web/src/App.less:27]: This was a wrong move, it was overwriting the color for the whole app 🥴

File: web/src/components/TestResults/TestResults.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
+import {Badge, Typography} from 'antd';
+import {useSelector} from 'react-redux';
```
Comment (xoscar) [inline web/src/components/TestResults/TestResults.tsx:2]: Moving it to its own component folder

File: web/src/components/Trace/Trace.tsx [hunk 4]
```diff
@@ -45,12 +45,7 @@ const Trace = ({displayError, visiblePortion, minHeight, test, testResultDetails
           <SpanDetail resultId={testResultDetails?.resultId} testId={test?.testId} span={selectedSpan} />
         </S.DetailsSection>
       </S.Main>
-      <TimelineDrawer
-        visiblePortion={visiblePortion}
-        testResultDetails={testResultDetails}
-        onSelectSpan={onSelectSpan}
-        selectedSpan={selectedSpan}
-      />
+      <TraceDrawer visiblePortion={visiblePortion} testResultDetails={testResultDetails} onSelectSpan={onSelectSpan} />
```
Comment (xoscar) [inline web/src/components/Trace/Trace.tsx:48]: Trace drawer is whatever we show at the bottom of the page

File: web/src/components/Trace/TraceDrawer.tsx [hunk 5]
```diff
@@ -1,33 +1,28 @@
...
-  selectedSpan,
-  testResultDetails,
-}: IProps): JSX.Element => {
+const TraceDrawer: React.FC<IProps> = ({visiblePortion, onSelectSpan, testResultDetails}) => {
   const [max, setMax] = useState(600);
+
   return (
     <ResizableDrawer open min={visiblePortion} max={max}>
-      <TraceTimeline
+      <TestResults
```
Comment (xoscar) [inline web/src/components/Trace/TraceDrawer.tsx:17]: Now we show the test results instead of the timeline chart.

**Review by kdhamric — APPROVED** [review]
Looks awesome!  If we can, lets push all the way through to the demo site - it is looking good.

**Review by cescoferraro — COMMENTED** [review]
I think the current component of TraceResult expands bigger the actual screen.
We need to take a look at this @xoscar

**Pull request merged**